src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Figure implements PrintableInfo, SetableArea {
+    private String name;
```

Comment:
Create additional enum for this field **OR** delete this field and write the name of the figure inside the method `draw()`